packages/kit/src/runtime/server/endpoint.js

```diff
@@ -40,9 +40,13 @@ export function is_text(content_type) {
  * @returns {Promise<Response | undefined>}
  */
 export async function render_endpoint(event, mod) {
+	const method = event.request.method.toLowerCase().replace('delete', 'del'); // 'delete' is a reserved word
```

Comment:
This code was already present, but I feel like `.replace()` isn't the best way to deal with this, for a few reasons. What if we just do something like:

```suggestion
	let method = event.request.method.toLowerCase();
	if (method === 'delete') { // 'delete' is a reserved word
		method = 'del';
	}
```

Reply:
moved it to a util as it was used in two places